src/huggingface_hub/file_download.py

```diff
@@ -487,6 +493,118 @@ def http_get(
         )
 
 
+def xet_get(
+    incomplete_path: Path,
```

Comment:
```suggestion
    *,
    incomplete_path: Path,
```

(nit) let's force keyword argument, easier to change things in the future